codespell_lib/data/dictionary.txt

```diff
@@ -20745,6 +20790,7 @@ simmilar->similar
 simpified->simplified
 simplei->simply
 simpley->simply
+simplication->simplification
```

Comment:
Does exist.
https://en.wiktionary.org/wiki/simplication

Reply:
I'll move it to rare.